test/SILOptimizer/moveonly_copyable_wrapper_capture.swift

```diff
@@ -0,0 +1,40 @@
+// RUN: %target-swift-frontend -emit-sil -verify %s
+
+class Class {}
+//struct Class : ~Copyable {}
```

Comment:
Sorta wondering why this struct is commented-out here.

Reply:
Oh, that's leftover from when I was reducing the problem, by swapping in a `~Copyable` type to see whether the problem reproduces with strictly-non-`Copyable` types. Turned out it only affects copyable types used as not-implicitly-copyable parameters.